app/src/main/java/com/mk/tasky/agenda/detail/components/DetailTimeSelector.kt

```diff
@@ -0,0 +1,96 @@
+package com.mk.tasky.agenda.detail.components
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.NavigateNext
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.mk.tasky.R
+import com.mk.tasky.ui.theme.Black
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+@Composable
+fun DetailTimeSelector(
+    text: String,
+    date: LocalDateTime,
+    isEditable: Boolean,
+    onTimeClick: () -> Unit,
+    onDateClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(top = 20.dp, bottom = 20.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Row(
+            modifier = Modifier.weight(9f),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Text(text = text, fontSize = 16.sp, color = Black)
+            Row(
+                modifier = Modifier.clickable {
+                    if (isEditable) {
+                        onTimeClick()
+                    }
+                }.padding(start = 10.dp, end = 10.dp),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(
+                    text = "${date.hour}:${date.minute}",
```

Comment:
Consider using a formatter here to ensure there's a leading zero when the hour is only one digit

Reply:
You are right!